pandas/tests/reshape/test_reshape.py

```diff
@@ -608,6 +608,70 @@ def test_get_dummies_all_sparse(self):
         )
         tm.assert_frame_equal(result, expected)
 
+    @pytest.mark.parametrize(
+        "values",
+        [
+            ["baz", "zoo"],
+            np.array(["baz", "zoo"]),
+            pd.Series(["baz", "zoo"]),
+            pd.Index(["baz", "zoo"]),
+        ],
+    )
+    @pytest.mark.parametrize("method", [True])
```

Comment:
are there any other method values to test?

Reply:
I also thought that method values might not be needed but I still left it. I was not sure how pytest worked and so didn't feel much confident. If you think it is redundant, I  will remove it.

Reply:
is there a `method=False` case that should be tested?  As it is, this test will be run 4 times (once for each `values` parameter above) and all 4 will have `method=True`.

Reply:
To the best of my knowledge there is no `method=False`. So, should I remove `method`?